src/view/ChewingEditor.cpp

```diff
@@ -223,8 +230,12 @@ void ChewingEditor::setupImport()
         model_, SIGNAL(importCompleted(bool, const QString&, size_t, size_t)),
         ui_.get()->notification, SLOT(notifyImportCompleted(bool, const QString&, size_t, size_t))
     );
+
+    // initialize import_count
+    import_count = 0;
```

Comment:
I use the word "initialize" instead of "reset" because the function "setupImport()" only run once -- when the program first start. IMO, "Reset" is more suitable referring to line 107.
Alternatively, I could have done this initialization in line 37, but I choose to initialize it here instead of in the constructor because i thought "import_count" is more related to the import functionality.